Karma/Enums.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Karma
+{
+    public class Enums
+    {
+        public enum Category
+        {
+            Clothes = 0,
+            Furniture,
+            Crafts,
+            Tools,
+            Books
+        }
```

Comment:
Are you planning to store all Enums in one file or to store different Enums in different files?

Reply:
We should also think about a convenient way of printing the enum in string to the user. Maybe using the 'Description' attribute? Refer to [https://stackoverflow.com/questions/630803/associating-enums-with-strings-in-c-sharp](url)

Reply:
> Are you planning to store all Enums in one file or to store different Enums in different files?

I'm thinking about storing them in one file.

Reply:
> We should also think about a convenient way of printing the enum in string to the user. Maybe using the 'Description' attribute? Refer to [https://stackoverflow.com/questions/630803/associating-enums-with-strings-in-c-sharp](url)

Users already see a string, not an integer that is stored in a database.

Reply:
Okay, then it's cool. Can't run your project because of different DB, so I didn't see how it looks.